src/Repository/Poll/NationalPollRepository.php

```diff
@@ -11,4 +12,16 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, NationalPoll::class);
     }
+
+    public function findLastActivePoll(): ?NationalPoll
+    {
+        return $this->createQueryBuilder('poll')
+            ->where('poll.finishAt > :now')
+            ->orderBy('poll.finishAt', 'desc')
+            ->setParameter('now', new Chronos())
```

Comment:
why do you use Chronos here?

Reply:
To test the case with no active national poll to return a local poll.